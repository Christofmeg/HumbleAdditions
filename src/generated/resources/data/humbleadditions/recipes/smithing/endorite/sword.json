{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorite_block"
  },
  "base": {
    "item": "humbleadditions:endorium_sword"
  },
  "result": {
    "item": "humbleadditions:endorite_sword"
  }
}