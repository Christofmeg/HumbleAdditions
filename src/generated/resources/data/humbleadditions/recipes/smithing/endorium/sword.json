{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorium_block"
  },
  "base": {
    "item": "minecraft:diamond_sword"
  },
  "result": {
    "item": "humbleadditions:endorium_sword"
  }
}