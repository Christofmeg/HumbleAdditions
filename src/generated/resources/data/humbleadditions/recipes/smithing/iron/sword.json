{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_ingot"
  },
  "base": {
    "item": "minecraft:golden_sword"
  },
  "result": {
    "item": "minecraft:iron_sword"
  }
}