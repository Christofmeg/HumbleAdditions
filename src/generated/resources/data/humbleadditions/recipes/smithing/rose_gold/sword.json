{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:rose_gold_ingot"
  },
  "base": {
    "item": "minecraft:golden_sword"
  },
  "result": {
    "item": "humbleadditions:rose_gold_sword"
  }
}