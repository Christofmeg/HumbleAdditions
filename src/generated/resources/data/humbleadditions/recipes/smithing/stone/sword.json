{
  "type": "minecraft:smithing",
  "addition": {
    "tag": "minecraft:stone_tool_materials"
  },
  "base": {
    "item": "minecraft:wooden_sword"
  },
  "result": {
    "item": "minecraft:stone_sword"
  }
}