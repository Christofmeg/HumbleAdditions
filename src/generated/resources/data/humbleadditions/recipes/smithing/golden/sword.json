{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_ingot"
  },
  "base": {
    "item": "minecraft:stone_sword"
  },
  "result": {
    "item": "minecraft:golden_sword"
  }
}